### PR TITLE
fix(mining-slot): safely handle zero VRF close window

### PR DESCRIPTION
**Pull request by blakebyrnes**
## Problem
The VRF-close window setting can be zero, and that value is used in close-check math. Without a guard, this can trigger divide-by-zero behavior in the bidding-close path.

## Fix
- add an explicit defensive guard in bidding-close evaluation when the close window is zero
- keep zero as a supported value to intentionally disable VRF-close behavior
- add tests for zero-window behavior and normal close-window paths

## Scope
- pallet-mining-slot logic and tests only
- no storage changes
- no migration

### CODE REVIEW

File: pallets/mining_slot/src/tests.rs [hunk 1]
```diff
@@ -1899,6 +1908,14 @@ fn it_doesnt_accept_bids_until_first_slot() {
 	});
 }
 
+#[test]
+fn it_allows_zero_vrf_close_window_update_to_disable_vrf_closing() {
+	new_test_ext().execute_with(|| {
+		assert_ok!(MiningSlots::configure_mining_slot_delay(RuntimeOrigin::root(), None, Some(0)));
+		assert_eq!(MiningConfig::<Test>::get().ticks_before_bid_end_for_vrf_close, 0);
+	});
```
[the remark this answers is not in the document]
Comment (blakebyrnes) [reply pallets/mining_slot/src/tests.rs:1932]: ok, fair. fixing
